gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -79,12 +79,15 @@ open class AnvilPlugin : Plugin<Project> {
     project.tasks
         .withType(KotlinCompile::class.java)
         .configureEach { compileTask ->
+          val shouldDisablePreciseJavaTracking = CheckMixedSourceSet(compileTask.project, compileTask)
+              .shouldDisablePreciseJavaTrackingIfNeeded()
           compileTask.doFirst {
             // Disable precise java tracking if needed. Note that the doFirst() action only runs
-            // if the task is not up to date. That's ideal, because if nothing needs to be
+             // if the task is not up to date. That's ideal, because if nothing needs to be
             // compiled, then we don't need to disable the flag.
-            CheckMixedSourceSet(compileTask.project, compileTask)
-                .disablePreciseJavaTrackingIfNeeded()
+            if(shouldDisablePreciseJavaTracking.get()) {
```

Comment:
Missing space. There are more KtLint violations: https://github.com/square/anvil/pull/114/checks?check_run_id=1099361039

Reply:
`./gradlew check` would be a good addition to `CONTRIBUTING.MD` ;)